src/store/dashboard/awsDashboard/awsDashboardWidgets.ts

```diff
@@ -94,6 +96,7 @@ export const databaseWidget: AwsDashboardWidget = {
     computedReportItem: ComputedReportItemType.cost,
     computedReportItemValue: ComputedReportItemValueType.total,
     titleKey: messages.dashboardCumulativeCostComparison,
+    chartName: 'dashboardCumulativeCostComparisonDatabase',
```

Comment:
Would you mind prefixing the chart names with "aws"; for example, "awsDatabaseChart"